pkg/controller/controller_test.go

```diff
@@ -3646,3 +3648,23 @@ func addGetSecretReaction(fakeKubeClient *clientgofake.Clientset, secret *corev1
 		return true, secret, nil
 	})
 }
+
+var dector cache.CacheMutationDetector
+var locker sync.Mutex
+
+func getMutationDetector() cache.CacheMutationDetector {
+	if dector == nil {
```

Comment:
Please fix the spelling of detector.

Reply:
@MHBauer 
I've update this. by the way, ci build always faild ,log is "8 The Kubernetes Authors.\n\nLicensed under the Apache License, Version 2.0 (the \"License\");\nyou may n", what' the meaning is ? is my core.autocrlf settings wrong ?

Reply:
I think needs rebase on https://github.com/kubernetes-incubator/service-catalog/pull/2541,  but it looks okay now?

Reply:
yes , i looks okay.